kubernetes/controllers/servicelevelobjective.go

```diff
@@ -19,17 +19,24 @@ package controllers
 import (
 	"context"
 	"fmt"
+	"time"
 
 	kitlog "github.com/go-kit/log"
 	"github.com/go-kit/log/level"
+	mimircli "github.com/grafana/mimir/pkg/mimirtool/client"
+	"github.com/grafana/mimir/pkg/mimirtool/rules/rwrulefmt"
```

Comment:
Given this simply wraps Prometheus' rulefmt. Do we really need that dependency? 

```go

// RuleGroup is a list of sequentially evaluated recording and alerting rules.
type RuleGroup struct {
	rulefmt.RuleGroup `yaml:",inline"`
	// RWConfigs is used by the remote write forwarding ruler
	RWConfigs []RemoteWriteConfig `yaml:"remote_write,omitempty"`
}
```

Reply:
done